src/wallet/db.cpp

```diff
@@ -387,7 +387,7 @@ bool CDB::Rewrite(const string& strFile, const char* pszSkip)
                         while (fSuccess) {
                             CDataStream ssKey(SER_DISK, CLIENT_VERSION);
                             CDataStream ssValue(SER_DISK, CLIENT_VERSION);
-                            int ret = db.ReadAtCursor(pcursor, ssKey, ssValue, DB_NEXT);
+                            ret = db.ReadAtCursor(pcursor, ssKey, ssValue, DB_NEXT);
```

Comment:
Technically speaking, this is not directly function scope.

But yes, one view is to have only one return value variable, the other is one return value variable per function call.

Renaming `ret` to `ret1` here could make this particular change smaller.